src/main/java/mate/academy/spring/controller/MovieSessionController.java

```diff
@@ -0,0 +1,61 @@
+package mate.academy.spring.controller;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.dto.moviesession.MovieSessionRequestDto;
+import mate.academy.spring.dto.moviesession.MovieSessionResponseDto;
+import mate.academy.spring.model.MovieSession;
+import mate.academy.spring.service.MovieSessionService;
+import mate.academy.spring.service.mapper.MovieSessionDtoMapper;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/movie-sessions")
+public class MovieSessionController {
+    private final MovieSessionService movieSessionService;
+    private final MovieSessionDtoMapper mapper;
+
+    public MovieSessionController(MovieSessionService movieSessionService,
+                                  MovieSessionDtoMapper mapper) {
+        this.movieSessionService = movieSessionService;
+        this.mapper = mapper;
+    }
+
+    @PostMapping
+    public MovieSessionResponseDto add(@RequestBody MovieSessionRequestDto movieSessionRequestDto) {
+        return mapper.toDto(movieSessionService.add(mapper.toModel(movieSessionRequestDto)));
+    }
+
+    @GetMapping("/available")
+    public List<MovieSessionResponseDto> findAllByIdAndDate(@RequestParam Long movieId,
+                                                            @RequestParam
+                                                            @DateTimeFormat(pattern = "dd.MM.yyyy")
+                                                            LocalDate date) {
+        return movieSessionService.findAvailableSessions(movieId, date).stream()
+                .map(mapper::toDto).collect(Collectors.toList());
```

Comment:
it's better to move "collect(Collectors.toList());" to a new line

Reply:
Agree, the convention (in **most** cases) tells us to start stream operations from the new line.
```suggestion
                .map(mapper::toDto)
                .collect(Collectors.toList());
```